packages/iov-bns/src/util.ts

```diff
@@ -12,17 +12,17 @@ import {
   TransactionKind,
 } from "@iov/bcp-types";
 import { Sha256 } from "@iov/crypto";
-import { Encoding } from "@iov/encoding";
+import { Bech32, Encoding } from "@iov/encoding";
 import { Algorithm, ChainId, PublicKeyBundle } from "@iov/tendermint-types";
 
-/** Encodes raw bytes into a printable address */
+/** Encodes raw bytes into a bech32 address */
 export function encodeBnsAddress(bytes: Uint8Array): Address {
-  return Encoding.toHex(bytes).toUpperCase() as Address;
+  return Bech32.encode("tiov", bytes).toUpperCase() as Address;
```

Comment:
sorry for the late info, but we should have all lower case, so `.toUpperCase()` should be removed